pkg/fanal/test/integration/testdata/goldens/packages/alpine-310.json.golden

```diff
@@ -3,131 +3,132 @@
     "ID": "alpine-baselayout@3.1.2-r0",
     "Name": "alpine-baselayout",
     "Version": "3.1.2-r0",
+    "Arch": "x86_64",
     "SrcName": "alpine-baselayout",
     "SrcVersion": "3.1.2-r0",
     "Licenses": [
       "GPL-2.0"
     ],
-    "Digest": "sha1:574d490311b68db01c0a3e44f5491be0cdc79250",
     "DependsOn": [
       "busybox@1.30.1-r2",
       "musl@1.1.22-r3"
     ],
     "Layer": {
-      "Digest": "sha256:9d48c3bd43c520dc2784e868a780e976b207cbf493eaff8c6596eb871cbd9609",
```

Comment:
It is weird. It is gone.

Reply:
Looks like it is old issue, but all contributors manually updated golden files.
I continue to investigate issue.

Reply:
I created #4379 to fix this problem.

Reply:
#4379 got merged.
@AliDatadog Could you merge the main branch and recreate the golden files again?

Reply:
Thanks it seems to work now 👍